README.md

```diff
@@ -58,84 +60,87 @@ Here is an example of using this module:
+| <a name="input_vpc_id"></a> [vpc_id](#input_vpc_id)                                                                                  | The ID of the VPC which the Tailscale Subnet Router EC2 instance will run in.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        | `string`       | n/a                                                                                                                                                                                                                                                                                                                                                                                                                                                               |   yes    |
 
 ## Outputs
 
-| Name | Description |
-|------|-------------|
-| <a name="output_autoscaling_group_id"></a> [autoscaling\_group\_id](#output\_autoscaling\_group\_id) | The ID of the Tailscale Subnet Router EC2 instance Autoscaling Group. |
-| <a name="output_instance_name"></a> [instance\_name](#output\_instance\_name) | The name tag value of the Tailscale Subnet Router EC2 instance. |
-| <a name="output_launch_template_id"></a> [launch\_template\_id](#output\_launch\_template\_id) | The ID of the Tailscale Subnet Router EC2 instance Launch Template. |
-| <a name="output_security_group_id"></a> [security\_group\_id](#output\_security\_group\_id) | The ID of the Tailscale Subnet Router EC2 instance Security Group. |
```

Comment:
was this also prettier? We should configure terraform-docs to output proper table or ignore this file in prettier to reduce churn.